pkg/runner/types.go

```diff
@@ -42,12 +44,23 @@ type Runner struct {
 	podMetrics  bool
 	cleanup     bool
 	serviceMesh bool
+	gatewayApi  bool
 	igNamespace string
 }
 
 type OptsFunctions func(r *Runner)
 
 var routesNamespace = benchmarkNs.Name
+var gatewayClassName string = "openshift-default"
+var gatewayNamespace gatewayv1beta1.Namespace = "openshift-ingress"
+var applicationNamespace gatewayv1beta1.Namespace = "ingress-perf"
+var portNumber gatewayv1beta1.PortNumber = 8080
+var tlsType gatewayv1beta1.TLSModeType = "Terminate"
+var fromNamespaces gatewayv1beta1.FromNamespaces = "All"
+var domainName = ex.Command("oc", "get", "ingresses.config/cluster", "-o", "jsonpath={.spec.domain}")
+var output, err = domainName.CombinedOutput()
```

Comment:
Grab this info using the API rather than oc

Reply:
It can be obtained from the metadata package of go-commons

https://github.com/cloud-bulldozer/go-commons/blob/main/ocp-metadata/ocp-metadata.go#L147-L162